NEWS.md

```diff
@@ -41,6 +41,9 @@
 * Improved usability of examples.
 * Renamed `random.cdisc.data.R` file to `package.R` and cleaned up contents.
 * Added more detailed documentation for function parameters.
+* Converted all dataset names to lowercase/snake case for consistency with other NEST packages.
+* Converted `ADSL` argument to `adsl`, `ADPC` to `adpc`, `event.descr` to `event_descr`, 
```

Comment:
hi @edelarua , can you please list out all the changes of upper cases to lower cases, this is potentially a big break change for users who actually use rcd package, i would like to make this very visible in the NEWS. Thanks!

Reply:
I've made a "Breaking changes" section at the top of NEWS to highlight the changes and expanded on which functions the changes apply to.